mep/people/tests/test_people_commands.py

```diff
@@ -16,10 +16,10 @@ def setUp(self):
     def test_get_queryset(self):
         # queryset should only include library members
         member = Person.objects.get(pk=189)  # francisque gay, member
-        author = Person.objects.get(pk=7152) # aeschylus, non-member
+        author = Person.objects.get(pk=7152)  # aeschylus, non-member
         qs = self.cmd.get_queryset()
-        self.assertTrue(member in qs)
-        self.assertFalse(author in qs)
+        assert member in qs
+        assert author not in qs
```

Comment:
do we not like unittest asserts?

Reply:
our convention for newer projects has been prefer the simpler pytest `assert` - more pythonic and more readable